From 28060b5f91207e70276be968073822a3dd06410a Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Tue, 30 May 2023 08:45:36 +0200
Subject: [PATCH 2/2] usb5807: backport to 5.15

Remove the devm_regulator... calls that aren't available in 5.15 yet
---
 drivers/usb/misc/usb5807.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/drivers/usb/misc/usb5807.c b/drivers/usb/misc/usb5807.c
index 5c5a543bf8c9..60864fa13f06 100644
--- a/drivers/usb/misc/usb5807.c
+++ b/drivers/usb/misc/usb5807.c
@@ -136,15 +136,6 @@ static int usb5807_i2c_probe(struct i2c_client *i2c)
 				     "Failed to request reset GPIO\n");
 	}
 
-	/* Enable power supplies while chip is held in reset */
-	ret = devm_regulator_get_enable(&i2c->dev, "vdd12");
-	if (ret)
-		return ret;
-
-	ret = devm_regulator_get_enable(&i2c->dev, "vdd33");
-	if (ret)
-		return ret;
-
 	/* Reset timing: Assert for >= 5 us */
 	usleep_range(5, 10);
 
-- 
2.17.1

